api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_educational_bookings.py

```diff
@@ -492,8 +492,6 @@ def create_industrial_educational_bookings() -> None:
                 name=f"[EAC] Lieux factice du CNL {i}",
```

Comment:
C'était déjà là avant, mais on pourrait peut-être corriger cette typo... sauf si c'est une _private joke_ EAC ?

```suggestion
                name=f"[EAC] Lieu factice du CNL {i}",
```

Reply:
Haha je ne pense pas, je corrige ça ! 
Et merci pour l'indication pour les remboursements, je crois qu'on a déjà un autre lieu avec un SIRET et des CB valides pour ces cas là 👍